website/content/docs/secrets/databases/postgresql.mdx

```diff
@@ -84,6 +84,70 @@ the proper permission, it can generate credentials.
     username           v-vaultuse-my-role-x
     ```
 
+## Rootless Configuration and Password Rotation for Static Roles
```

Comment:
I am wondering if we should make this more generic and put it on a separate page? Maybe on the [Overview](https://developer.hashicorp.com/vault/docs/secrets/databases) page? For now we only support Postgres and Oracle. But if/when more DB's support the feature it will become redundant to add the steps to every plugin since the steps are all the same. There is nothing specific to Postgres (or any DB) about the setup.

If we do put anything on the DB specific pages, we may consider providing the minimum set of privileges required for the static role to manage its own password. 

Thoughts?

Reply:
Had a similar thought on adding to the Overview page but decided to keep the documentation scoped to Postgres for the 1.18 cycle 😄 

We should be good on minimum privileges for Postgres too since by default users are allowed to rotate their own passwords.